R/prep_scores.R

```diff
@@ -1,7 +1,87 @@
-prep_scores <- function(results_portfolio = NULL, peers_results_aggregated = NULL,
-                        asset_class = "equity") {
-  if (is.null(results_portfolio)) {
-    data_out <- use_toy_data("scores") %>% filter(asset_class == .env$asset_class)
+#' Prepare data input for plotting aggregate climate scores
+#'
+#' @description Prepare data input for plotting aggregate climate scores based
```

Comment:
nit: I believe that you don't need the tag `@description`, it will be automatically inferred.

Reply:
done